src/_guides/openapi/specification/v3.1/cheatsheet.md

```diff
@@ -0,0 +1,25 @@
+---
+title: OpenAPI 3.1 - The Cheat Sheet
+authors: chris
+excerpt: Everything you need to keep in mind when writing an OpenAPI contract, on a one-pager.
+date: 2024-09-12
+---
+
+Now that you've roamed through the Complete Guide, let's make long stories short. We have listed the key elements you should always keep in mind when writing a new OpenAPI contract, or maintaining existing ones.
+
+![openapi-31-cheatsheet-v1.png](https://storage.googleapis.com/bump-blog-resources/openapi-31-cheasheet/openapi-31-cheatsheet-v1.png)
+[Download the PDF version](https://storage.googleapis.com/bump-blog-resources/openapi-31-cheasheet/openapi-31-cheatsheet-v1.pdf)
+
+The Cheat Sheet is presented here in an initial version.
+
+Here are the current sections:
+* Document Structure
+* General Information
+* API Structure
+* Data Types and Schemas
+* Security
+* Reuse Elements
+* Polymorphism
+* Grouping and sorting
+
+For any feedback and suggestions, please open an issue [on this site's GitHub Repository](https://github.com/bump-sh/docs/issues). We are currently working on building a web version of the Cheat Sheet, so that anyone can directly contribute to it with a Pull Request.
```

Comment:
```suggestion
For any feedback and suggestions, please open an issue [on our GitHub Repository](https://github.com/bump-sh/docs/issues). We are currently working on building a web version of the Cheat Sheet, so that anyone can directly contribute to it with a Pull Request.
```